### PR TITLE
Adding colors and breakpoints from theme

### PR DESCRIPTION
**Pull request by gp231982**
Adding colors and breakpoints from theme

### CODE REVIEW

**Review by GracjanaKromska — APPROVED** [review]
Ok, thanks for adding colors and breakpoints from theme in your components. 
That was important.

File: src/features/MovieList/styled.js
Comment (KamilJustynski) [inline src/features/MovieList/styled.js:1]: Please remove the margin left and right here

File: src/common/PersonTile/styled.js
Comment (KamilJustynski) [inline src/common/PersonTile/styled.js:1]: Here the color should not be black but woodSmoke

**Review by KamilJustynski — CHANGES_REQUESTED** [review]
Please correct these things

File: src/common/PersonTile/styled.js
Comment (gp231982) [reply src/common/PersonTile/styled.js:1]: I changed it for woodSmoke

Comment (KamilJustynski) [reply src/common/PersonTile/styled.js:1]: Its okej! Thanks Grzegorz

**Review by KamilJustynski — APPROVED** [review]
Its okej! Thanks Grzegorz

**Pull request merged**